### ISSUE TITLE
Question about input shape of ZeroPadding*D

### ISSUE THREAD

**Issue by t-ae** [tensorflow/swift-apis#494]
`ZeroPadding1D` takes 1D tensor:

https://github.com/tensorflow/swift-apis/blob/74a9bd2f7332a117bb159653bec865a00e9d3745/Tests/TensorFlowTests/LayerTests.swift#L246-L252

But [`tf.keras.layers.ZeroPadding1D`](https://www.tensorflow.org/api_docs/python/tf/keras/layers/ZeroPadding1D) takes 3D tensor that has shape `(batch, axis_to_pad, features)`.

Is this difference intentional?

Same for `ZeroPadding2D`and `ZeroPadding3D`.

**Comment by Shashi456** [comment]
@t-ae Not intentional, When I wrote the layers, I wrote them directly with dimensionality in my mind i.e 1D, 2D, 3D. I think we'll have to update the functionality of zero padding layers wrt keras.

**Comment by Shashi456** [comment]
@t-ae now that i've checked the implementation, i've understood that the implementation is sorta right, we just need to add the batch and features dimensions to everything
